index.js

```diff
@@ -82,6 +82,8 @@ function FSWatcher(_opts) {
   // Enable fsevents on OS X when polling isn't explicitly enabled.
   if (undef('useFsEvents')) opts.useFsEvents = !opts.usePolling;
 
+  FsEventsHandler.init(opts.requireFseventsCallback);
```

Comment:
May as well put `if (opts.useFsEvents)` in front of this. It will add the advantage of sparing some initialization time for everyone else.